pkg/cri/sbserver/container_stats_list_linux.go

```diff
@@ -29,13 +29,55 @@ import (
 
 	containerstore "github.com/containerd/containerd/pkg/cri/store/container"
 	"github.com/containerd/containerd/pkg/cri/store/stats"
+	containerstorestats "github.com/containerd/containerd/pkg/cri/store/stats"
 )
 
 func (c *criService) containerMetrics(
 	meta containerstore.Metadata,
 	stats *types.Metric,
 ) (*runtime.ContainerStats, error) {
 	var cs runtime.ContainerStats
+	rawMetrics, err := c.rawContainerMetrics(meta, stats)
```

Comment:
I think we need analogous changes in https://github.com/containerd/containerd/blob/main/pkg/cri/server/container_stats_list_linux.go#L33

(looks like the sbserver has been forked, so we need to update in both places)